mix/mix_message.nim

```diff
@@ -2,15 +2,12 @@ import chronicles, results
 import ./[config, protocol, utils]
 
 type MixMessage* = object
-  message: seq[byte]
-  protocol: ProtocolType
+  message*: seq[byte]
+  protocol*: ProtocolType
```

Comment:
felt weird having these fields be private, but have a getter that got all teh fields